ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -664,15 +667,15 @@ def postprocessoutput(self, outputfiles, outputdir):
                 if outputfileClass in backend_output_postprocess[backendClass]:
 
                     if not self.subjobs:
-                        logger.info("Job %s Setting Location of %s: %s" % (self.getFQID('.'), getName(outputfile), outputfile.namePattern))
```

Comment:
To me it looks like you get DEBUG for the master and INFO for the subjob with this change. Don't you want it the other wayr around? Other places seems to have similar logic.

Reply:
`not self.subjobs` implies the job doesn't have subjobs, so it is itself a subjob. I agree it is confusing. I'll change it to `if self.master`

Reply:
Actually I think it is fine as is - from my tests it works as intended.